environment-dev.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
   #
   # optional
   - joblib
+  - lxml
```

Comment:
pandas `read_html` needed this (or bs4)

Reply:
sorry didn't know that was there.  it was just in my environment

Reply:
No prob, I think it's fine to leave things this way, since the pandas message is reasonably clear.